arrows/core/mesh_intersect.cxx

```diff
@@ -0,0 +1,385 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief Implementation of mesh operations
+ */
```

Comment:
Can we use the preferred documentation style for new code, please?
```suggestion
/// \file
/// Implementation of mesh operations.
```

(The guide recommends to omit `\brief` for *member* documentation; not sure we've been doing that for file documentation also, or if AUTOBRIEF works for those, so I'd be okay leaving a `\brief` here. But we should probably add a period and use `///` at least.)

Reply:
I will make sure to re-read the code style documentation. Is there header/*.cxx that we know has the preferred documentation style.